apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -861,6 +876,19 @@ export default function teacherSections(state = initialState, action) {
       }
     }
 
+    let newSections = _.omit(state.sections, oldSectionId);
+    newSections[section.id] = {
+      ...state.sections[section.id],
+      ...section
+    };
+
+    let newStudentSectionIds = Object.values(newSections)
+      ?.filter(section => section.participantType === 'student')
+      .map(section => section.id);
+    let newPlSectionIds = Object.values(newSections)
+      ?.filter(section => section.participantType !== 'student')
+      .map(section => section.id);
```

Comment:
I think you can do without `?.` here and above -- `Object.values` appears to either return an array or throw an exception.